include/boost/geometry/algorithms/detail/partition.hpp

```diff
@@ -629,8 +608,7 @@ class partition
                                        IteratorVector& iterator_vector,
                                        ExpandPolicy const& expand_policy)
     {
-        for(typename boost::range_iterator<ForwardRange const>::type
-                it = boost::begin(forward_range);
+        for (auto it = boost::begin(forward_range);
             it != boost::end(forward_range);
             ++it)
```

Comment:
Would it make sense to put the loop in one line? It should be lesser than 100 characters.